README.md

```diff
@@ -13,7 +13,7 @@
 
 A collection of simplified and community-driven man pages.
 
-Install it with `npm install -g tldr` or [try the web client](http://tldr.ostera.io).
+Install it with `npm install -g tldr`, `brew install tldr` (on OSX, using [Homebrew](http://brew.sh)), or [try the web client](http://tldr.ostera.io).
```

Comment:
Yeah, I could see this saying something like:

> Install it with `npm install -g tldr`, or [try the web client](http://tldr.ostera.io). On OS X, a client is also available using `brew install tldr`.

but as @agnivade said, it's a little awkward to mention one platform-specific installer and not mention others, like yum or apt-get. And if we do mention all those, we'd make this opening sentence too bulky, and downplay the other clients below, so yeah, I guess I'd also agree it's better just not to mention it here.